src/app/services/raiden.service.ts

```diff
@@ -646,6 +647,76 @@ export class RaidenService {
         );
     }
 
+    openBatchOfChannels(
+        token: UserToken,
+        connectionChoices: ConnectionChoice[]
+    ): Observable<void> {
+        let notificationIdentifier: number;
+        let errorCount = 0;
+
+        return of(null).pipe(
```

Comment:
Nitpick: you could replace `of(constant)+tap` with a `defer` call, if it's just to ensure a function is called at subscription time.

Reply:
Nice! I didn't know about it.